Sources/StreamChatUI/ChatMessageList/RETHINK/MessageContentView.swift

```diff
@@ -240,17 +276,21 @@ class MessageContentView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
 
         // Reactions
         if options.contains(.reactions) {
+            let reactionsBubbleView = createReactionsBubbleView()
+            addSubview(reactionsBubbleView)
+
             let reactionsView = createReactionsView()
-            reactionsView.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
-            reactionsView.setContentCompressionResistancePriority(.defaultHigh, for: .vertical)
-            addSubview(reactionsView)
+            reactionsBubbleView.addSubview(reactionsView)
+            reactionsView.pin(to: reactionsBubbleView.layoutMarginsGuide)
```

Comment:
Previously this setup was inside the bubble but I tired to flatten this a bit. To me bubble view by its own looks weird since it's not clear how it should be used. However, the flattened structure is easier to follow then a nested `bubble -> content` one